### ISSUE TITLE
Can't open file for writing

### ISSUE THREAD

**Issue by Guppy16** [dracula/solidworks#2]
Hi,

I followed these [steps](https://draculatheme.com/solidworks) to install Dracula Solidworks using MSYS 2. However, when I run `./override_background.sh` the error msg below shows. I continued with the steps and found that only some of the Dracula theme works as shown in the screenshot below. Please could someone help? 

```
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll" [converted][unix format] 558876 lines, 84748366 bytes
Error detected while processing command line:
E486: Pattern not found: 73 *, *73 *, *73
E486: Pattern not found: 80 *, *80 *, *80
E486: Pattern not found: 55 *, *55 *, *55
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll"
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll" E212: Can't open file for writing
Press ENTER or type command to continue
```

![image](https://user-images.githubusercontent.com/48368985/135302585-6cc41714-ec0d-43a8-85d2-bd9599410e79.png)

---EDIT---

- I have checked that the dll file exists as per this [stackoverflow post](https://stackoverflow.com/questions/8253362/etc-apt-sources-list-e212-cant-open-file-for-writing)
- I have also checked that I can open the file in MSYS using vim

**Comment by eleanor-clifford** [comment]
Hi, I've pushed an update that should fix E486, but I think it would only have occured if you've run `./override_background.sh` multiple times - is this the case? (it was a bug regardless, just checking for my sanity)

As for E212, I'm a bit confused because it looks like vim has successfully edited the DLL judging by the screenshot. Since it worked fine I don't think it's a breaking error, but can you replace `swStyleBlueu.dll` with the original (which should have been backed up in `solidworks-filebackup` and try again to see if it still shows up?

The rest of the theme is set by points 2 and 3 in the instructions rather than `./override_background.sh`, have you followed those yet? If you have done, can you send me a screenshot of *Tools->Options->Colors*?

**Comment by Guppy16** [comment]
Yes, I ran the script multiple times. E486 makes sense then. Just checking, does that mean that the backup `swStyleBlueu.dll` was overridden with the edited one after I reran it multiple times?

After replacing the dll with the backup the colours still looked the same - unless the dll was overridden, what is the purpose of changing the dll? Is it possible to copy your dll?

I then also tried re-running the script, but it gave the same E212 error:
```
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll" [converted][unix format] 558876 lines, 84748366 bytes
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll"
Error detected while processing command line:
"/c/Program Files/SOLIDWORKS Corp/SOLIDWORKS/swStyleBlueu.dll" E212: Can't open file for writing
Press ENTER or type command to continue
```

I have complete the rest of the steps. Here's the screenshot of _Tools->Options->Colors_:

![image](https://user-images.githubusercontent.com/48368985/135312197-9c0b93be-bc37-44c1-ad40-7a6930df1d01.png)

**Comment by eleanor-clifford** [comment]
Ah yes the backup would have been overriden by the edited one. That's unfortunate. I'll add a check to fix that in future, but I'm not too worried about E212 anyway since the colors seem to be working fine (though I can send you a fresh dll to try again with if you'd like). Incidentally, there are two reasons why this repo edits the dll instead of providing a working one: robustness to differences in different versions of SW, and licensing.

And ah, the background appearance option needs to be set to *Gradient*. I'll add that to the instructions.

**Comment by Guppy16** [comment]
Ok, I solved the issue. Turns out that I needed to run msys as administrator. Windows sees msys as an app and doesn't give an option to run as administrator, so I had to navigate to the exe and run it as administrator from there. 

I believe the error was caused because this line didn't execute properly:
`chmod +w /c/Program\ Files/SOLIDWORKS\ Corp/SOLIDWORKS/swStyleBlueu.dll` 
(running it separately gave a `Permission Denied` error.)

However, it doesn't seem that the background colour is right - my first screenshot shows the background colour is white (and still is), but the background colour in the image on the [tutorial](https://draculatheme.com/solidworks) is darker. Not sure if the css needs to be modified or smth.

Thanks for the help 👍
